internal/server/server.go

```diff
@@ -87,7 +89,13 @@ func StartUp(Version, LoadFileType string) {
 	conf.InitConf()
 	factory.InitClientsFactory()
 
-	err := store.SetupWithKuiperConfig(conf.Config)
+	undo, err := maxprocs.Set(maxprocs.Logger(conf.Log.Infof))
+	if err != nil {
+		panic(err)
```

Comment:
Do we really need to panic here? We can continue without it, right?

Reply:
Done.